### DISCUSSION TITLE
Storing book series by different authors together.

### DISCUSSION THREAD

**Discussion by Don-Cal** [advplyr/audiobookshelf#2768] [Ideas]
The current library directory structure is AUTHOR\SERIES\XX - BOOKTITLE 

There are several series where there are different authors for different books in the series. 
The current structure means that books in a series are stored with different authors.

Could users have the option to create a separate series library where the folder structure is SERIES\AUTHOR\XX - BOOKTITLE?

Users could then keep their series with multiple authors together.

**Comment by pwinnski** [comment]
Books in a series are all together in the ABS interface, different authors or not. 

AUTHOR1/SERIES1/XX - BOOKTITLE
AUTHOR2/SERIES1/XX - BOOKTITLE

These are listed together in ABS. 

If you want to store them that way on disk, I will just note that the only truly hard-and-fast rule of folder structures is that each audiobook must be in a separate folder. So you could certainly put your files in a SERES/AUTHOR structure today, it's just that the scanner would set the author names to SERIES and the series name to AUTHOR on first import. If you then edited the metadata, either manually or via matching, I believe the end result would be what you want.

**Reply by Don-Cal** [reply]
Thanks for the response.
Im well aware how things currently work, which is why Im asking for the option to store books the way I did.

I prefer keeping the series together as I am frequently out of wifi range and streaming the audiobook keeps stopping and starting which gets annoying. I'm using Audiobookshelf as a library I can browse and then copy the series to my phone manually using Android File Manager or using a wired connection 

ABS does not provide the option to download a series of books at once, you have to do it slowly, one at a time.
Also having the series folders under different authors could result in using different series names in different locations.
Ie Before ABS I used to denote a series under the author by appending "Series" to the series folder name OR some series folders could be missing A, THE, etc prefixes.

**Reply by nichwall** [reply]
App version 0.9.77-beta added the ability to download an entire series at once. This is downloading the series defined within ABS and is independent of how they are stored in the file system.

https://github.com/advplyr/audiobookshelf-app/releases/tag/v0.9.77-beta

**Reply by Don-Cal** [reply]
Awesome. 
I look forward to testing it out when I get home in a couple weeks. 

Thanks for the info.
